app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
```

Comment:
=========================== short test summary info ============================
FAILED tests/test_main.py::test_car_wash_station[cars1-wash_station1-22.4] - ...
FAILED tests/test_main.py::test_car_wash_station[cars3-wash_station3-40.3] - ...
FAILED tests/test_main.py::test_car_wash_station[cars4-wash_station4-13.0] - ...
========================= 3 failed, 17 passed in 0.16s =========================

you fail 3 test